### PR TITLE
update welcome page view

### PR DESCRIPTION
**Pull request by YouQam**

### CODE REVIEW

**Comment by YouQam** [comment]
> Please fix tests before review. Thank you.

Working on it.

File: src/styles/oeb/typography.scss [hunk 1]
```diff
@@ -24,3 +24,9 @@
 	background: var(--color-red);
 	color: #fff;
 }
+
+.oeb-purble-bg {
```
Comment (umut0) [inline src/styles/oeb/typography.scss:28]: purple

**Review by umut0 — COMMENTED** [review]
Looks good. I could not find mobile views for this screen, but i think you solution is fine. can merge. Maybe just change purble to purple to avoid confusion.

File: src/styles/oeb/typography.scss (hunk 1 above)
Comment (YouQam) [reply src/styles/oeb/typography.scss]: Already updated.

**Comment by umut0** [comment]
One more thing: The texts are hard coded now? Did we not want to use the translations anymore? If not we can merge.

**Review by umut0 — APPROVED** [review]

**Pull request merged**